packages/CollabCodeEdit-Server.package/CCEServer.class/instance/CCEheartbeat.from..st

```diff
@@ -0,0 +1,4 @@
+request handling
+CCEheartbeat: anOrderedDictionary from: aCCERemoteClient
+	" this delay solves the problem, that the server imediatly drops the connection when the message arrives after estblishing the connection"
```

Comment:
I love comments!

Reply:
but this one is necessary to understand the futureSend

Reply:
Thats why i love it!